app/waf/test/fixtures/datasources/testrest.datasource.ts

```diff
@@ -2186,6 +2187,33 @@ Reason:
   },
 
   installTrustedExtensions200: () => {},
+
+  bigiqLoginSucceed200: () => {
+    return {
+      token: {
+        token: 'abcdefgh',
+        exp: new Date().getTime() / 1000 + 300,
+      },
+    };
+  },
+
+  bigiqRevokeLicenseSucceed202: () => {
+    return {
+      id: '12345678',
+    };
+  },
+
+  bigiqRevokeTaskFailed200: () => {
+    return {
+      status: 'FAILED',
+    };
+  },
+
+  bigiqRevokeTaskFinished200: () => {
+    return {
+      status: 'FINISHED',
+    };
+  },
```

Comment:
Please post the real responses here.

Reply:
Other properties are useless. We needn't to add them.